mitmproxy/options.py

```diff
@@ -30,19 +30,19 @@ def __init__(
             app_port: int = APP_PORT,
             anticache: bool = False,
             anticomp: bool = False,
-            client_replay: Sequence[str] = (),
+            client_replay: Sequence[str] = [],
```

Comment:
This is intended. Options returns a deepcopy of all values to guard against accidental mutation. We need the empty default to compare equal to empty values returned from other parts of our code. This leaves us either with finicky and error-prone list->tuple conversions throughout the code, or trying to special case sequence comparisons. I think just returning copies and specifying mutable default arguments is the least worst option here.